packages/DatePicker/src/helpers.js

```diff
@@ -1,13 +1,20 @@
 import React from "react";
+import momentPropTypes from "react-moment-proptypes";
 
-export function extractChildrenProps(children) {
+export function extractChildrenProps(children, target) {
   let overrides;
 
   React.Children.forEach(children, child => {
-    if (child.type.displayName === "DatePicker.Input") {
+    if (child.type.displayName === target.displayName) {
       overrides = { ...child.props };
     }
   });
 
   return overrides;
 }
+
+export function isMomentObjectOrNull(date) {
+  if (date === null) return true;
+
+  return momentPropTypes.momentObj(date);
+}
```

Comment:
I think this means we need to bump the version for this `helpers` package in its `package.json`... right @nahumzs?